data/json/items/gunmod/muzzle.json

```diff
@@ -54,7 +54,50 @@
     "dispersion_modifier": 15,
     "handling_modifier": 1,
     "loudness_modifier": -30,
-    "flags": [ "DISABLE_SIGHTS" ]
+    "flags": [ "DISABLE_SIGHTS", "CONSUMABLE" ]
+  },
+  {
+    "id": "filter_suppressor",
+    "type": "GUNMOD",
+    "name": "'solvent trap'",
+    "description": "A once /completely legal/ thread adapter attached to an automotive filter with a hole punched in the end, generating an illegal, unregistered silencer. Good thing there don't seem to be any ATF agents around to arrest you. While close in effectivness to a real suppressor, it is not designed around any particular caliber and will eventually lose effectiveness. The installed filter is large and, when attached, will interfere with your ability to aim down the base sights of the gun.",
+    "weight": 880,
+    "volume": 3,
+    "price": 480,
+    "to_hit": 1,
+    "bashing": 3,
+    "material": [ "steel" ],
+    "symbol": ":",
+    "color": "dark_gray",
+    "location": "muzzle",
+    "mod_targets": [ "pistol", "smg", "rifle" ],
+    "consume_chance": 100,
+    "consume_divisor": 40,
+    "damage_modifier": -5,
+    "dispersion_modifier": 10,
+    "handling_modifier": 1,
+    "loudness_modifier": -20,
+    "flags": [ "DISABLE_SIGHTS", "CONSUMABLE" ]
+  },
+  {
+    "id": "bottle_suppressor",
+    "type": "GUNMOD",
+    "name": "soda bottle silencer",
+    "description": "This is a 'silencer' made from a 2 liter soda bottle, some smaller bottles for baffles, and a generous length of duct tape.   Gunfire is extremely loud and can damage your hearing without protection and you can be certain that this will not save your ears whatsover. What it might do is allow you to get a shot or two off with a slightly quieter report.  This simple suppressor is large and, when attached, will interfere with your ability to aim down the base sights of the gun.",
```

Comment:
```suggestion
    "description": "This is a 'silencer' made from a 2 liter soda bottle, some smaller bottles for baffles, and a generous length of duct tape.  Gunfire is extremely loud and can damage your hearing without protection and you can be certain that this will not save your ears whatsoever.  What it might do is allow you to get a shot or two off with a slightly quieter report.  This simple suppressor is large and, when attached, will interfere with your ability to aim down the base sights of the gun.",
```